index.js

```diff
@@ -13,3 +13,143 @@ hamburg.addEventListener('click', toggleMenu);
+  myModal.classList.remove('show');
+}
+
+/* eslint-disable */
+
+function onClickProject(index) {
+  const project = projects[index];
+
+  const projectHTML = `
+  <div class="popup">
+  <div class="popup-header">
+      <h5 class="popup-name">${project.name}</h5>
+      <button class="popup-btn-close">
+        <img src="./image/x-icon.png" alt="" />
+      </button>
+    </div>
+    <ul class="popup-details">
+      <li class="detail1">${project.details[0]}</li>
+      <img src="./image/Counter.png" alt="" />
+      <li class="detail2">${project.details[1]}</li>
+      <img src="./image/Counter.png" alt="" />
+      <li class="detail3">${project.details[2]}</li>
+    </ul>
+    <div class="popup-img-container">
+      <img src="${project.img}" alt="Project Image" />
+    </div>
+    <div class="popup-img-container-desktop">
+      <img src="${project.img2}" alt="Project Image" />
+    </div>
+    <div class="popup-body">
+      <div class="popup-desc-container-mobile">
+      <p>Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent</p>
+      </div>
+      <div class="popup-desc-container-desktop">
+      <p>${project.description}</p>
+      </div>
+      <div class="tech-and-buttons">
+      <div class="popup-technologies-buttons">
+        <ul class="popup-technologies">
+          <li class="popup-technology1">${project.technologies[0]}</li>
+          <li class="popup-technology2">${project.technologies[1]}</li>
+          <li class="popup-technology3">${project.technologies[2]}</li>
+        </ul>
+      </div>
+      <div class="popup-tech-buttons-desk">
+        <ul class="popup-technologies">
+          <li class="popup-technology1">${project.technologies2[0]}</li>
+          <li class="popup-technology2">${project.technologies2[1]}</li>
+          <li class="popup-technology3">${project.technologies2[2]}</li>
+        </ul>
+        <ul class="popup-technologies">
+          <li class="popup-technology4">${project.technologies2[3]}</li>
+          <li class="popup-technology5">${project.technologies2[4]}</li>
+        </ul>
+      </div>
+      <div class="popup-buttons">
+        <a href="${project.liveLink}" class="popup-button1">
+        See Live
+          <img src="./image/button1.png" alt="Icono"/>
+        </a>
+        <a href="${project.fontLink}" class="popup-button2">
+        See source
+          <img src="./image/button2.png" alt="Icono" />
+        </a>
+      </div>
+      
+      </div>
+    </div>
+  
+  
+  </div>
+  `;
+
+  myModal.innerHTML = projectHTML;
+  showPopup();
+
+  const closeButton = document.querySelector(".popup-btn-close");
+  closeButton.addEventListener("click", closePopup);
+}
```

Comment:
Nice job implementing the functionalities of your popup😉. But we still have some improvements to make the popup match the given design😉.

on the desktop screen, kindly consider adding the exact spacings (padding or margin) around each element both on mobile and desktop👍.

your popup
<img width="941" alt="image" src="https://github.com/rodrigopenayo98/my-rod-porf/assets/64643995/45d14aec-934e-4188-816d-cf0d98a69428">


design
<img width="353" alt="image" src="https://github.com/rodrigopenayo98/my-rod-porf/assets/64643995/ba66aacb-ee74-4c6c-908e-7a64f9db8b94">

kindly also correctly position the marked vector icon as it is overlapping making it look different from the design😉.
<img width="942" alt="image" src="https://github.com/rodrigopenayo98/my-rod-porf/assets/64643995/723a8901-f177-4ff4-b90b-3c28c5d1411c">



yours on mobile

<img width="350" alt="image" src="https://github.com/rodrigopenayo98/my-rod-porf/assets/64643995/0e101fd8-b218-47a9-83ce-665d40fa0a5c">

design
<img width="104" alt="image" src="https://github.com/rodrigopenayo98/my-rod-porf/assets/64643995/4e7468df-6747-4745-8e16-a3dab6bd7380">
Kindly add the necessary spacing around the popup so it becomes easy to see it👍.

don't forget you are also required to add the button interactions for the different states as we are required to stick to the given design😉.

<img width="224" alt="image" src="https://github.com/rodrigopenayo98/my-rod-porf/assets/64643995/708071a1-9606-495a-ae38-7738861afd12">

kindly implement the different button states as stated in the design👍.